include/manifold/manifold.h

```diff
@@ -412,9 +413,9 @@ class Manifold {
   Manifold RefineToTolerance(double) const;
   Manifold SmoothByNormals(int normalIdx) const;
   Manifold SmoothOut(double minSharpAngle = 60, double minSmoothness = 0) const;
-  static Manifold Smooth(const MeshGL&,
+  static Manifold Smooth(MeshGL,
```

Comment:
Looking at the code, it seems that we will need to copy it because we changed the `faceID` array. I think a better approach would be to keep the old API, do the copy internally, and try to avoid that copying later, e.g. by making another `MeshGL` that uses `VecView` or something.